udrzba/_pomocne.php

```diff
@@ -21,13 +31,13 @@ function nasad(array $nastaveni) {
       /dokumentace
 
       /nastaveni/*
-      !/nastaveni/' . $nastaveni['souborNastaveni'] . '
+      !/nastaveni/{$nastaveni['souborNastaveni']}
       !/nastaveni/nastaveni.php
       !/nastaveni/nastaveni-vychozi.php
       !/nastaveni/zavadec.php
       !/nastaveni/zavadec-zaklad.php
 
-      /testy
+      /tests
```

Comment:
Tohle byla moje chyba v předchozím pull requestu, kdy jsem přejmenoval adresář s testy na více konvenční název, ale nezměnil jsem to tady v ignore, takže se začal kopírovat při deploy na server.